### ISSUE TITLE
something wrong with running the blackbird

### ISSUE THREAD

**Issue by yanbosmu** [butor/blackbird#294]
Is this how blackbird works?
JUST creating a log file?

I run blackbird in an ubuntu VPS on vultr.com 

Blackbird Bitcoin Arbitrage
DISCLAIMER: USE THE SOFTWARE AT YOUR OWN RISK

Log file generated: output/blackbird_log_20171002_104107.log
Blackbird is running... (pid 29451)

**Comment by greatwolf** [comment]
Yes, it doesn't print anything to stdout besides the initial message. Most of the events and status of the bot is recorded in the log instead. You can track it to a terminal output using something like the `tail` command though.
